pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -190,6 +190,23 @@ class RichProgressBarTheme:
     metrics: Union[str, Style] = "white"
 
 
+def _detect_light_colab_theme() -> bool:
+    """Detect if it's light theme in Colab."""
```

Comment:
Did you find this function somewhere or did you write it yourself? If the former, please add a reference

Reply:
https://github.com/googlecolab/colabtools/issues/2485